src/scss/_objects.file-dropzone.scss

```diff
@@ -0,0 +1,45 @@
+.grommetux-dropzone {
```

Comment:
I added this due to difficulties getting the scss to compile in grommet-core.  It should be removed before merging.  I will be submitting the PR for the dropzone very soon.